tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     pretend
     pytest
     ./vectors
+passenv = ARCHFLAGS LDFLAGS CFLAGS INCLUDE LIB LD_LIBRARY_PATH
```

Comment:
We don't use it in any of our testing right now, so we probably can leave it out.